File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int number;
    // ask for the number till the user inputs a valid number.
    do
    {
        number = get_int("Enter an int: ");
    }
    while (number > 23 || number < 0);

    // Do this on every line
    for (int i = 0; i < number; i++)
    {
        // prints the initial spaces to make it look like a pyramid.
        for (int k = 0; k < number - i - 1; k++)
        {
            printf(" ");
        }
        // print the first half of the pyramid.
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // print two spaces in the middle of the pyramid.
        printf("  ");
        // prints the second half of the pyramid.
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/sentimental/mario/more/mario.py
from cs50 import get_int

# ask for the number till the user inputs a valid number.
while True:
    number = get_int("Enter an int: ")
    if (number < 24 and number > - 1):
        break

# Do this on every line
for i in range(number):
    # prints the initial spaces to make it look like a half pyramid.
    for k in range(number - i - 1):
        print(" ", end="")

    # print the first half of the pyramid.
    for j in range(i + 1):
        print("#", end="")

    print("  ", end="")

    # prints the second half of the pyramid.
    for j in range(i + 1):
        print("#", end="")

    # prints a line at the end of every line.
    print("\n", end="")
